### PR TITLE
[UA] Love Domain Cleric

### PR DESCRIPTION
**Pull request by FreshRoyal**
Original iteration of the Unity Domain that was leaked.

### CODE REVIEW

File: unearthed-arcana/2020/20200206.xml [hunk 1]
```diff
@@ -566,4 +566,138 @@
...
+			<grant type="Spell" id="ID_PHB_SPELL_HOLD_MONSTER" level="9" spellcasting="Cleric" prepared="true" />
+		</rules>
+	</element>
+	<element name="Emboldening Bond" type="Archetype Feature" source="Unearthed Arcana: Subclasses, Part 2" id="ID_WOTC_UA20200206_ARCHETYPE_FEATURE_LOVE_EMBOLDENING_BOND">
+		<description>
+			<p><em>1st-level Love Domain feature</em></p>
+			<p>You can forge an empowering bond between allies. As an action, you can choose two willing creatures you can see within 30 feet of you (this can include yourself) and create a magical bond between them. While either bonded creature is within 30 feet of the other, the creature can roll a d4 and add the number rolled to an attack roll, an ability check, or a saving throw it makes. Each creature can add the d4 no more than once per turn. The bond lasts for 1 hour or until you use this feature again.</p>
+			<p class="indent">You can use this feature once, and you regain the ability to do so when you finish a long rest. You can also expend a spell slot to use the feature again.</p>
+		</description>
+		<sheet action="Action">
```
Comment (AngryGuineaPig) [inline unearthed-arcana/2020/20200206.xml:651]:
```suggestion
		<sheet action="Action" usage="1/Long Rest">
```

**Comment by AngryGuineaPig** [comment]
The URL in the Source needs to be changed to
https://web.archive.org/web/20200204140440/https://media.wizards.com/2020/dnd/downloads/UA2020-Subclasses02.pdf

**Comment by FreshRoyal** [comment]
> The URL in the Source needs to be changed to https://web.archive.org/web/20200204140440/https://media.wizards.com/2020/dnd/downloads/UA2020-Subclasses02.pdf

Done

**Review by AngryGuineaPig — APPROVED** [review]

File: unearthed-arcana/2020/20200206.xml [hunk 2]
```diff
@@ -566,4 +566,156 @@
...
+	<element name="Protective Bond" type="Archetype Feature" source="Unearthed Arcana: Subclasses, Part 2" id="ID_WOTC_UA20200206_ARCHETYPE_FEATURE_LOVE_PROTECTIVE_BOND">
+		<description>
+			<p><em>6th-level Love Domain feature</em></p>
+			<p>The bond you forge between people shields them from harm. While either creature chosen for your Emboldening Bond feature is within 30 feet of the other, the creature can use its reaction to grant resistance to all damage to the other creature when that other creature takes damage. This resistance lasts until the end of the current turn.</p>
+		</description>
+		<setters>
+			<set name="sourceUrl">https://web.archive.org/web/20200204140440/https://media.wizards.com/2020/dnd/downloads/UA2020-Subclasses02.pdf</set>
+		</setters>
+		<sheet>
+			<description>While either creature chosen for your Emboldening Bond feature is within 30 feet of the other, the creature can use its reaction to grant resistance to all damage to the other creature when that other creature takes damage. This resistance lasts until the end of the current turn.</description>
```
Comment (NickVendel) [inline unearthed-arcana/2020/20200206.xml:683]:
```suggestion
			<description>While either creature chosen for your Emboldening Bond feature is within 30 feet of the other, the creature can use its reaction to grant resistance to all damage to the other creature when that other creature takes damage. This resistance lasts until the end of the current turn.</description>
			<description level="17">While creatures chosen for your Emboldening Bond feature are on the same plane of existence, either creature can use its reaction to grant resistance to all damage to the other creature when that other creature takes damage. This resistance lasts until the end of the current turn.</description>
```

File: unearthed-arcana/2020/20200206.xml [hunk 3]
```diff
@@ -566,4 +566,156 @@
...
+		</description>
+		<setters>
+			<set name="sourceUrl">https://web.archive.org/web/20200204140440/https://media.wizards.com/2020/dnd/downloads/UA2020-Subclasses02.pdf</set>
+		</setters>
+		<sheet>
+			<description>Creatures affected by your Emboldening Bond feature gain its benefits, as well as those of your Protective Bond feature, while they are on the same plane of existence as each other.
+			When a creature chosen for your Emboldening Bond is reduced to 0 hp, their bonded partner gains the following benefits for 1 minute, or until the creature regains at least 1 hp: &#13;
+			• The creature has advantage on attack rolls, ability checks, and saving throws &#13;
+			• The creature gains resistance to all damage &#13;
+			• As an action, the creature can touch their bonded partner to expend and roll any number of Hit Dice. Their bonded partner regains a number of hp equal to the total rolled.</description>
```
Comment (NickVendel) [inline unearthed-arcana/2020/20200206.xml:718]:
```suggestion
			<description>When a creature chosen for your Emboldening Bond is reduced to 0 hp, their bonded partner gains the following benefits for 1 minute, or until the creature regains at least 1 hp: &#13;
			• The creature has advantage on attack rolls, ability checks, and saving throws &#13;
			• The creature gains resistance to all damage &#13;
			• As an action, the creature can touch their bonded partner to expend and roll any number of Hit Dice. Their bonded partner regains a number of hit points equal to the total rolled.</description>
```

File: unearthed-arcana/2020/20200206.xml [hunk 4]
```diff
@@ -566,4 +566,156 @@
...
+		</sheet>
+	</element>
 </elements>
```
Comment (NickVendel) [inline unearthed-arcana/2020/20200206.xml:721]:
```suggestion
</elements>

```

**Review by NickVendel — CHANGES_REQUESTED** [review]

**Review by MeriadarGoblin — APPROVED** [review]

**Review by NickVendel — APPROVED** [review]

**Pull request merged**